### PR TITLE
Fix wrong check for logfile in admin summary

### PR DESCRIPTION
**Pull request by nupplaphil**
Fixes https://github.com/friendica/friendica/issues/7676#issuecomment-546498267

### CODE REVIEW

File: src/Module/Admin/Summary.php [hunk 1]
```diff
@@ -85,20 +86,27 @@ public static function content()
...
+				if (!empty($file)) {
+					$stream = $fileSystem->createStream($file);
+
+					if (!isset($stream)) {
+						throw new InternalServerErrorException('Stream is null.');
+					}
+				}
+
+			} catch (\Throwable $exception) {
+				$warningtext[] = L10n::t('The developer logfile \'%s\' is not usable. No logging possible (error: \'%s\')', $file, $exception->getMessage());
```
Comment (tobiasd) [inline src/Module/Admin/Summary.php]: I don't think that "logging" is a *developer* tool. I would just remove that word from the warning.

Comment (MrPetovan) [reply src/Module/Admin/Summary.php]: Why not?

**Comment by AlfredSK** [comment]
I'm not a developer and I'm using it too.

File: src/Module/Admin/Summary.php (hunk 1 above)
Comment (nupplaphil) [reply src/Module/Admin/Summary.php]: It's not the "default" logfile, which is possible to set in the admin-panel and its called `dlogfile`, so I thought d stands for developer :-)

how about "non-default" or "custom" or "technical" logfile?

Comment (MrPetovan) [reply src/Module/Admin/Summary.php]: It stand for "debug" if I remember correctly.

**Pull request merged**